db/migrate/20160617123557_add_meta_fields_to_theming.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetaFieldsToTheming < ActiveRecord::Migration
+  def change
+    add_column :pageflow_themings, :default_author, :string
+    add_column :pageflow_themings, :default_publisher, :string
+    add_column :pageflow_themings, :default_keywords, :string
```

Comment:
I'm also talking about HTML5 placeholders. My suggestion is to display the default values from the config as placeholders in the input fields, since - given the current fallback logic - these actually are the values which are used if the entry specific meta attributes are blank, right? Same thing in the theming form.